examples/go-uretprobe-counter/main.go

```diff
@@ -1,4 +1,5 @@
-//go:build amd64
+//go:build linux
+// +build linux
```

Comment:
hmm why changing only this file not the rest of the examples ?

Reply:
Made this like the rest. Not sure why it was different from the others originally, but it was causing multi-arch not to build for just uretprobe.